SketchUp/Sketchup/Face.rb

```diff
@@ -279,18 +277,18 @@ def followme(*args)
   #   tw = Sketchup.create_texture_writer
   #   uvHelp = face.get_UVHelper(true, true, tw)
   #
-  # @param front
+  # @param front [Boolean]
   #   True if you want the texture coordinates for the front
   #   face, false if not. Defaults to true.
   #
-  # @param back
+  # @param back [Boolean]
   #   True if you want the texture coordinates for the back
   #   face, false if not. Defaults to true.
   #
-  # @param texturewriter
-  #   A TextureWriter object.
+  # @param texturewriter [Sketchup::TextureWriter]
+  #   An optional TextureWriter object.
```

Comment:
If you change the signature to `def get_UVHelper(front, back, texturewriter = nil)` then YARD will automatically indicate it as optional.